internal/host/validator.go

```diff
@@ -395,6 +396,42 @@ func (v *validator) printHostnameUnique(c *validationContext, status validationS
 	}
 }
 
+func (v *validator) doesTimeSkewExists(c *validationContext) validationStatus {
+	if c.inventory == nil {
+		return ValidationPending
+	}
+	if c.inventory.Timestamp == 0 {
+		return ValidationPending
+	}
+	for _, h := range c.cluster.Hosts {
+		if h.ID.String() != c.host.ID.String() && h.Inventory != "" && *h.Status != models.HostStatusDisconnected {
+			var otherInventory models.Inventory
+			if err := json.Unmarshal([]byte(h.Inventory), &otherInventory); err != nil {
+				v.log.WithError(err).Warnf("Illegal inventory for host %s", h.ID.String())
+				continue
+			}
+
+			if math.Abs(float64(c.inventory.Timestamp-otherInventory.Timestamp))/60 > float64(v.hwValidatorCfg.MaximumAllowedTimeDiffMinutes) {
```

Comment:
If there is one host with time skew, all the hosts will fail this validation.  Is that what we want?

Reply:
it is what was agreed for now